k8s/oidc-vault/k8s/server-statefulset.yaml

```diff
@@ -69,11 +69,6 @@ spec:
           - name: spire-data
             mountPath: /run/spire/data
             readOnly: false
-          readinessProbe:
```

Comment:
What do you think about doing an HTTP probe to the "/keys" endpoint instead? Will that work? That seems better than nothing.